utils/src/main/java/se/fortnox/reactivewizard/util/rx/RetryWithDelayFlux.java

```diff
@@ -0,0 +1,46 @@
+package se.fortnox.reactivewizard.util.rx;
+
+import reactor.core.publisher.Flux;
+
+import java.time.Duration;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public class RetryWithDelayFlux implements Function<Flux<Throwable>, Flux<?>> {
```

Comment:
move into reactor-client module for now, to avoid getting Flux dependencies in utils module at this stage?